extensions/nametags/ace_nametags.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * ace_nametags.cpp
+ * Author: esteldunedain
+ * Implements nametags using Intercept
+ *
+ * Takes:
+ * None
+ *
+ * Returns:
+ * None
+ */
+#include <Windows.h>
+#include <stdio.h>
+#include <cstdint>
+#include <atomic>
+#include <memory>
+
+#include "intercept.hpp"
+#include "logging.hpp"
+#include "client\client.hpp"
+#include "client\pointers.hpp"
+#include "client\sqf\uncategorized.hpp"
+#include "client\sqf\config.hpp"
+
+#include "nametagger.hpp"
+
+#define PI 3.14159265f
```

Comment:
Sure. But I think I'm not even using PI or any of the math functions. It's just copy/pasted crap.